snapcraft_legacy/cli/version.py

```diff
@@ -28,7 +30,8 @@ def versioncli():
 
 
 @versioncli.command("version")
-def version():
+@add_verbosity_options()
+def version(**kwargs):
```

Comment:
Some commands like this one don't get called, but it's easier to just make the changes everywhere.